ASTRO/ASTROpdd.tex

```diff
@@ -0,0 +1,264 @@
+
+\IEEEPARstart{T}{he} RIT Astro Tracking team formed in the Fall of 2016 and was designed for the purpose of Astronomical research and the tracking of artificial satellites.  Considerable growth has been made in regard to general Astronomy skills with a number of quality images and the improvement of general astrophotography skills.  Clear pictures of Jupiter, Saturn, and other objects have been taken by members and showcased at various SPEX related events.  In addition the team has developed software for the automation of its primary telescope, the Meade LX200, where it can be controlled from any location with wireless connectivity.
+
+The team has accomplished much since its inception and the next logical step would be to improve the skills developed as well as begin preliminary tracking of both artificial satellites and other celestial objects.  In order to begin new members will firstly have to be educated on basics, technology to assist in tracking must be developed, and technology must be developed for the maintenance of current equipment.
+
+
+\section{Primary Objective}
+\label{sec:primary-obj}
+  % At the end of the day, whether the project ``succeeds'' or ``fails'' is judged against the objectives it sought to meet.
+  % Note that results that contradict expectations/hypotheses are not failures if the scientific \& engineering methods are followed along the way.
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+
+    \subsection{Tracking Mount}
+    \label{subsec:trackingmount}
+        This project involves the research and development of an active tracking mount for taking pictures with a DSLR of celestial objects.  In order to get pictures of stars the camera needs to take in a lot of light.  However, stars move very fast and will appear as blurred lines if the shutter is not fixed on them for the duration of a long exposure shot.  The mount allows the camera to move along with the stars and capture more light which results in photos otherwise not to be seen with the naked eye.
+    \subsection{Orion Mount}
+    \label{subsec:orionmount}
+        This project involves the research and development of a new mount for the Orion telescope.  It is up to the team to determine a style for the mount and create said mount.  It would also be of preference to have this later able to be automated.
+    \subsection{Tracking Software}
+    \label{subsec:trackingsoftware}
+        This project involves creating software to handle the equations to convert two line elements into a format the telescope understand.  This will involve converting the equations to Cartesian form and plotting the orbits in a 3-Dimensional space.  Will then need to be interfaced with the telescopes
```

Comment:
wip?

Reply:
Sorry what is wip?  "What is purpose" or "work in progress"?

Reply:
work in progress

Reply:
Sweet!  Thank you!